data-loader/core/src/main/java/com/scalar/db/dataloader/core/util/CollectionUtil.java

```diff
@@ -0,0 +1,19 @@
+package com.scalar.db.dataloader.core.util;
+
+import java.util.Collection;
+
+/** Utils for collection classes */
+public class CollectionUtil {
+
+  /**
+   * Check if lists are of same length
+   *
+   * @param collections List of collections
+   * @return collections are same length or not
+   */
+  public static boolean areSameLength(Collection<?>... collections) {
+    int N = collections[0].size();
+    for (Collection<?> a : collections) if (a.size() != N) return false;
+    return true;
```

Comment:
```suggestion
    int n = collections[0].size();
    for (Collection<?> a : collections) {
      if (a.size() != n) {
        return false;
      }
    }
    return true;
```

Reply:
I have updated the change as suggested.
Thank you.